sopel/modules/meetbot.py

```diff
@@ -391,7 +443,16 @@ def meetinginfo(bot, trigger):
 def log_meeting(bot, trigger):
     if not ismeetingrunning(trigger.sender):
         return
-    if trigger.startswith('.endmeeting') or trigger.startswith('.chairs') or trigger.startswith('.action') or trigger.startswith('.info') or trigger.startswith('.startmeeting') or trigger.startswith('.agreed') or trigger.startswith('.link') or trigger.startswith('.subject'):
+    if (
+            trigger.startswith('.endmeeting') or
+            trigger.startswith('.chairs') or
+            trigger.startswith('.action') or
+            trigger.startswith('.info') or
+            trigger.startswith('.startmeeting') or
+            trigger.startswith('.agreed') or
+            trigger.startswith('.link') or
+            trigger.startswith('.subject')
+    ):
```

Comment:
This whole if block is bullshit if the bot's `prefix` isn't `.`. IIRC it should be checking `trigger.group(1)`, which should contain the command name that was triggered without the prefix.

Reply:
The only groups available seem to both be the entire message. Possibly since this isn't a `@command`?